gateway/package.json

```diff
@@ -0,0 +1,107 @@
+{
+  "name": "bos-workspace-gateway",
+  "version": "0.0.0",
+  "packageManager": "npm",
+  "dependencies": {
+    "@braintree/sanitize-url": "^6.0.2",
```

Comment:
This may be related to VM versioning -- there was a patch recently regarding sanitize URL.

See the VM version and how sanitizeUrl is removed from dependencies in most recent [NearSocial/viewer](https://github.com/NearSocial/viewer/blob/master/package.json)